msrs/models.py

```diff
@@ -52,7 +54,7 @@ class Msrs(models.Model):
     gender = models.CharField(max_length=15, blank=True, choices=GENDER_CHOICES)
     whatsapp = models.CharField("Whatsapp", max_length=11)
     state = models.CharField("Estado", max_length=2)
-    city = models.CharField("Cidade", max_length=100)
+    city = models.ForeignKey(Cities, on_delete=models.CASCADE, verbose_name="Cidade")
```

Comment:
colocaria o `on_delete` como protect, pq não queremos perder esse dado de cidade das msrs

```suggestion
    city = models.ForeignKey(Cities, on_delete=models.PROTECT, verbose_name="Cidade")
```